packages/embark/src/cmd/cmd_controller.js

```diff
@@ -97,7 +98,8 @@ class EmbarkController {
       webServerConfig: webServerConfig,
       webpackConfigName: options.webpackConfigName,
       singleUseAuthToken: options.singleUseAuthToken,
-      ipcRole: 'server'
+      ipcRole: 'server',
+      pkg: PKG
```

Comment:
Do you really need to pass it to engine? Since it's in the config, it should be sufficient, no?

Reply:
I think it's only in the config when `initConfig` is called, which doesn't happen when the `Engine` is instantiated IIRC, but could definitely be wrong.